packages/amplify-provider-awscloudformation/src/iterative-deployment/deployment-manager.ts

```diff
@@ -46,39 +56,46 @@ export class DeploymentError extends Error {
 }
 
 /**
- *
+ * Deployment operation type
  */
 export type DeploymentOp = Omit<DeploymentMachineOp, 'region' | 'stackTemplatePath' | 'stackTemplateUrl'> & {
   stackTemplatePathOrUrl: string;
 };
 
 /**
- *
+ * Deployment step type
  */
 export type DeploymentStep = {
   deployment: DeploymentOp;
   rollback: DeploymentOp;
 };
 
+type EventMap = {
```

Comment:
this doesn't look like map. what is this type representing? what kind of event is this?